lib/src/room.dart

```diff
@@ -793,7 +807,11 @@ class KurentoRoom extends ChangeNotifier implements Room {
         break;
       // Handles the case when the Agent leaves the room on a call transfer, name is turned to null to represent that agent is no longer in the call
       case 'LEFT':
-        _agentName = null;
+        _agentsName.remove(agentId.toString());
+        final tracks = _tracksByAgentId[agentId];
+        tracks?.forEach((trackId) async {
```

Comment:
Nit picking here. But I would say to prefer `for` loop over `forEach` functions for several reasons but especially when using async closures. One thing to note in this case is that `forEach` doesn't care about what the closure returns so if you return future it will not wait for it. Might not be relevant if you don't care about waiting for each item to finish executing.

Reply:
First I didn't care about being sequence sensitive but know that you mentioned this I would like to be more cautions on this.

Reply:
Thanks!